src/ray/util/event.h

```diff
@@ -187,8 +192,13 @@ class RayEventContext final {
 // for sending
 class RayEvent {
  public:
-  RayEvent(rpc::Event_Severity severity, const std::string &label)
-      : severity_(severity), label_(label) {}
+  RayEvent(rpc::Event_Severity severity, RayLogLevel log_severity,
```

Comment:
maybe worth add comments to mention we require `file_name` to be a string literal which has static storage duration; otherwise we might have memory issues.
cc @mwtian